lib/adiwg/mdtranslator/readers/iso19115_2/modules/module_citation.rb

```diff
@@ -15,6 +16,7 @@ module Citation
                @@titleXPath = 'gmd:title//gco:CharacterString'
                @@dateXPath = 'gmd:date'
                @@responsibilityXPath = 'gmd:citedResponsibleParty'
+               @@onlineResourceXPath = './/gmd:onlineResource'
```

Comment:
bummer that you have to break the mold here and search lower than the immediate child but i get it. `onlineResource` exists as an immediate child to `CI_Citation` in -3 but not in -2. `onlineResource` is also a key in the `newCitation` hash.